assets/cubyz/sbb/oak/1/trunk_6.zig.zon

```diff
@@ -0,0 +1,21 @@
+.{
+	.blueprint = "cubyz:oak/1/trunk_6",
+	.children = .{
+		.blue = .{
+			.{.structure = "cubyz:oak/1/branch_1", .chance = 0.333},
+			.{.structure = "cubyz:oak/1/branch_2", .chance = 0.333},
+			.{.structure = "cubyz:oak/1/branch_3", .chance = 0.334},
```

Comment:
Note that the chances don't need to add up to 1, if you want all paths have the same chance, you can set it to `1` (which is the default, so you can just omit it)
```suggestion
			.{.structure = "cubyz:oak/1/branch_1"},
			.{.structure = "cubyz:oak/1/branch_2"},
			.{.structure = "cubyz:oak/1/branch_3"},
```